tests/acceptance/test_conftest.py

```diff
@@ -22,17 +22,8 @@
 
```

Comment:
Why the filename is `test_conftest` rather than `test_login` or `test_navigation_page` ?